playbooks/common/openshift-cluster/upgrades/etcd/backup.yml

```diff
@@ -3,6 +3,7 @@
   vars:
     embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
     timestamp: "{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
+    etcdctl_command: "{{ 'etcdctl' if not openshift.common.is_containerized else 'docker exec etcd_container etcdctl' }}"
```

Comment:
Yeah, it's still problematic to use etcdctl from the host on embedded containerized installs because the versioning can be different but I think this does the best thing for all the "supported" scenarios and is no worse than status quo for embedded containerized etcd instances.